test/python/transpiler/test_mappers.py

```diff
@@ -281,11 +281,12 @@ class TestsLookaheadSwap(SwapperCommonTestCases, QiskitTestCase):
     pass_class = LookaheadSwap
 
 
-class TestsStochasticSwap(SwapperCommonTestCases, QiskitTestCase):
-    """Test SwapperCommonTestCases using StochasticSwap."""
+### Commented out as StochasticSwap is deprecated
+# class TestsStochasticSwap(SwapperCommonTestCases, QiskitTestCase):
+#     """Test SwapperCommonTestCases using StochasticSwap."""
 
-    pass_class = StochasticSwap
-    additional_args = {"seed": 0}
+#     pass_class = StochasticSwap
+#     additional_args = {"seed": 0}
```

Comment:
I am not too convinced by having this test commented out. In principle we should be testing deprecated classes up until their removal (when tests make sense). I think that instead, you could try filtering out the deprecation warning in the `CommonUtilitiesMixin` class, where the passes are instantiated and appended to the pass manager (L108). It is possible to only filter out warnings that match a specific message. You can see an example of this filtering [here](https://github.com/Qiskit/qiskit/blob/59630272672e4593e2bd5570680e47961d398923/qiskit/providers/basic_provider/basic_simulator.py#L239-L246).

Reply:
I agree, I asked Luciano about this case and he said it was fine, but turns out the fix wasn't so hard... once someone tells you exactly what you should do!